API/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "whitethoothapi",
+  "version": "1.0.0",
+  "description": "Mock API for teams with multi environment support",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run build && node ./build/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc"
+  },
+  "author": "PizzaTip",
+  "license": "MIT",
+  "dependencies": {
+    "@types/express": "^4.17.2",
+    "express": "^4.17.1",
+    "typescript": "^3.7.5"
```

Comment:
for knowledge, TS is actually a package?

Reply:
Yap